### ISSUE TITLE
loss too high, mIoU too low

### ISSUE THREAD

**Issue by Alohomora0130** [Tramac/Fast-SCNN-pytorch#52]
Hello author, may I ask what is the learning rate and batch_size you are using? I ran the code directly, and its loss fluctuated between 1.1 and 0.9. I also used other learning rates, but the effect was not very good. Can you share me with your learning rate and batch_size? Thank you very much!
